index.js

```diff
@@ -41,6 +71,8 @@ function run() {
   });
 
   if (changelog_was_updated) {
+    echo('...verifying changelog');
+    verifyChangelog();
```

Comment:
Wrap this in a try-catch. If it throws, you can call `revertChanges()` and output something to indicate what happens. It's reasonable to use one exit code to cover all these cases.

Alternatively, we could leave the changes and just not commit them, in which case we should direct the user to manually verify the changes, and run `git checkout -- CHANGELOG.md` if they want to revert.